src/sentry/api/endpoints/organization_events.py

```diff
@@ -137,6 +137,30 @@ def get(self, request, organization):
         )
 
 
+class OrganizationEventsTagsEndpoint(OrganizationEventsEndpointBase):
+    def get(self, request, organization):
+        try:
+            snuba_args = self.get_snuba_query_args(request, organization)
+        except OrganizationEventsError as exc:
+            return Response({'detail': exc.message}, status=400)
+        except NoProjects:
+            return Response({'count': 0})
+
+        keys = request.GET.get('keys')
+        if not keys:
+            return Response({'detail': 'Event tag keys must be specified'}, status=400)
+        snuba_args['filter_keys']['tags_key'] = keys
```

Comment:
Does snuba only expect string values tag key values? Should we ensure that the user hasn't sent us lists, or dictionaries?

Reply:
lol mark. :) you always review in progress prs.